testproxy/services/bulk-mutate-rows.js

```diff
@@ -39,16 +39,36 @@ const bulkMutateRows = ({clientMap}) =>
     } catch (error) {
       if (error.name === 'PartialFailureError') {
         return {
-          status: error,
+          status: {
```

Comment:
does this still need to be a special case? It looks the same as the other statement?